extensions/twitch/src/helpers/streamlink.ts

```diff
@@ -8,6 +8,15 @@ export function watchStream(
   lowlatency: boolean | undefined,
   streamlinkConfig: string | undefined,
 ) {
+  if (!streamlinkLocation || streamlinkLocation === "") {
```

Comment:
in JS `""` is falsy, which means it will be included in the `!streamlinkLocation` check
```suggestion
  if (!streamlinkLocation) {
```